### PR TITLE
feat: Add new privacy policy page

### PR DESCRIPTION
**Pull request by pnu-s**
Add a new page for storing the Android app privacy policy for https://github.com/Exodus-Privacy/exodus-android-app/issues/111

The new page will be hosted at: https://exodus-privacy.eu.org/en/page/privacy-policy/

**Screenshot:**

image

### CODE REVIEW

File: content/page/privacy-policy.en.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+## Privacy Policy (Android)
+
+The εxodus Android application does not collect any personal information.
+
+The application requires the following permissions:
+
+* *INTERNET*: This permission is required so that the application can query the reports on the main εxodus instance <https://reports.exodus-privacy.eu.org/>. It is not used for any other purpose.
+* *ACCESS_NETWORK_STATE*: This permission allows us to know whether you are connected to the Internet before making any request.
```
Comment (U039b) [inline content/page/privacy-policy.en.md:14]: I think we should replace `us` by `the application`

File: content/page/privacy-policy.en.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The application transmits the list of installed applications on the device, mixed with a number of random applications (not present on the device) in order to anonymize the list. This list is not kept by the εxodus server.
+
+On the εxodus API we keep some information for 16 days, for abuse prevention: IP address, date, time, requested URL and user-agent.
```
Comment (U039b) [inline content/page/privacy-policy.en.md:18]: I propose: `On the εxodus API server, we [...]`

**Review by U039b — CHANGES_REQUESTED** [review]

File: content/page/privacy-policy.en.md (hunk 2 above)
Comment (jfoucry) [reply content/page/privacy-policy.en.md:18]: Did we have a french version?
Did we need a french version?

Comment (jfoucry) [reply content/page/privacy-policy.en.md:18]: εxodus is the platform, Exodus is the app and Exodus Privacy is the organistion. So the first pharse should be:
`The Exodus Applicaiton` no?

**Comment by pnu-s** [comment]
>  Did we need a french version?

@jfoucry For the time being I did not push a french version, I don't think so we absolutely **need** one for the time being, because the app will link to the english version anyway.

**Review by codeurimpulsif — APPROVED** [review]
Ok for me

**Pull request merged**